packages/matter-node.js/src/log/FileLogger.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright 2022-2023 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Level } from "@project-chip/matter.js/log";
+import { createWriteStream } from "fs";
```

Comment:
It would be probably be better to use async I/O for this although if node buffers internally it might not matter.  NodeJsActionTracer.ts in device-gen is an example of using async I/O with a public API that's synchronous

Reply:
adjusted